internal/state/mailbox_search.go

```diff
@@ -4,425 +4,581 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/ProtonMail/gluon/internal/db"
+	"github.com/bradenaw/juniper/parallel"
+	"github.com/bradenaw/juniper/xslices"
 	"net/mail"
+	"runtime"
 	"strings"
+	"sync/atomic"
```

Comment:
might wanna `goimports` this, shouldn't be mixing stdlib and extern imports

Reply:
Done